assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/daemonset.yaml

```diff
@@ -133,13 +133,15 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            # This env var enables calico to expose prometheus metrics
+            # Note: This is not enabled in upstream typhoon by default
```

Comment:
Why mention typhoon?

Reply:
Because the configs are verbatim that ship in typhoon.

Reply:
I would remove that, I don't think we should highlight the relation with typhoon in any way.

Reply:
I agree with @invidian

Reply:
done